### PR TITLE
*: add six documents of TiDB Lightning

### PR DESCRIPTION
**Pull request by lilin90**
Via: https://github.com/pingcap/docs-cn/pull/976

### CODE REVIEW

File: tools/lightning-monitor.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+- **`lightning_idle_workers`** (Gauge)
+
+    Counting idle workers. Values should be less than the `table-concurrency`/`region-concurrency` settings and are typically zero. Labels:
+
+    - **name**: `table` / `region`
+
+- **`lightning_kv_encoder`** (Counter)
+
+    Counting open and closed KV encoders. KV encoders are in-memory TiDB instances which converts SQL `INSERT` statements into KV pairs. The net values need to be bounded in a healthy situation. Labels:
```
Comment (CaitinChen) [inline tools/lightning-monitor.md]:
```suggestion
    Counting open and closed KV encoders. KV encoders are in-memory TiDB instances which convert SQL `INSERT` statements into KV pairs. The net values need to be bounded in a healthy situation. Labels:
```

File: tools/lightning-overview-architecture.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+# TiDB Lightning Introduction and Architecture
+
+TiDB Lightning is a tool used for fast full import of large amounts of data into a TiDB cluster. Currently, TiDB Lightning supports reading SQL dump exported via mydumper. You can use it in the following two scenarios:
+
+- Importing **large amounts** of **new** data **quickly**
+- Back up and restore all the data
+
+## Lightning Architecture
```
Comment (CaitinChen) [inline tools/lightning-overview-architecture.md]:
```suggestion
## Lightning architecture
```

File: tools/lightning-faq.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+It is potentially caused by starting `tidb-lightning` incorrectly, which causes the system to send a SIGHUP signal to stop it. If this is the case, there should be a log entry like:
+
+```
+2018/08/10 07:29:08.310 main.go:47: [info] Got signal hangup to exit.
+```
+
+We do not recommend using `nohup` directly in the command line. Rather, put the `nohup` inside a script file and execute the script.
+
+## Why my TiDB cluster is using lots of CPU and running very slowly after using Lightning?
```
Comment (CaitinChen) [inline tools/lightning-faq.md]:
```suggestion
## Why my TiDB cluster is using lots of CPU resources and running very slowly after using Lightning?
```

File: tools/lightning-faq.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+## Can one `tikv-importer` serve multiple `tidb-lightning` instances?
+
+Yes, as long as every `tidb-lightning` instance operates on different tables.
+
+## How to stop `tikv-importer`?
+
+If it is deployed using TiDB-Ansible, run `scripts/stop_importer.sh` under the deployed folder.
+
+Otherwise, obtain the process ID with `ps aux | grep tikv-importer`, then run `kill «pid»`.
```
Comment (CaitinChen) [inline tools/lightning-faq.md]:
```suggestion
Otherwise, obtain the process ID with `ps aux | grep tikv-importer`, and then run `kill «pid»`.
```

**Comment by lilin90** [comment]
@CaitinChen Thanks! All comments addressed. PTAL again.
@kennytm @GregoryIan PTAL

File: tools/lightning-checkpoints.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
+---
+title: TiDB Lightning Checkpoints
+summary: Use checkpoints to avoid redoing the previously completed tasks before the crash.
+category: tools
+---
+
+# TiDB Lightning Checkpoints
```
Comment (kennytm) [inline tools/lightning-checkpoints.md]:
```suggestion
# TiDB-Lightning Checkpoints
```

According to https://github.com/pingcap/tidb-lightning/pull/80#discussion_r232165326 we should hyphenate it

**Comment by lilin90** [comment]
@kennytm Thanks! PTAL again

File: media/tidb-lightning.svg [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+      <g>
+         <text style="stroke:none" x="166.4921547" y="178" font-family="Helvetica" font-size="8" font-weight="normal" font-style="normal">KV </text>
+         <text style="stroke:none" x="163.158170325" y="188" font-family="Helvetica" font-size="8" font-weight="normal" font-style="normal">pairs</text>
+      </g>
+      <path style="fill:#FFFFFF" id="Paper" d="M61.9776804448,91 L39.3157993472,91 L39.3157993472,127 L69.5557993472,127 L69.5557993472,98.5781189024 L61.9776804448,91 L60.9776804448,91 L61.4776804448,91 L61.4776804448,98.5781189024 L69.0557993472,98.5781189024"/>
+      <g>
+         <text style="stroke:none" x="46.0783774722" y="106" font-family="Helvetica" font-size="8" font-weight="normal" font-style="normal">SQL </text>
+         <text style="stroke:none" x="43.9299399722" y="116" font-family="Helvetica" font-size="8" font-weight="normal" font-style="normal">dump</text>
+      </g>
+      <text style="stroke:none" x="5.5" y="230.730876623" font-family="Helvetica" font-size="12" font-weight="normal" font-style="italic">TiDB Lightning Tool Set</text>
```
Comment (kennytm) [inline media/tidb-lightning.svg]:
```suggestion
      <text style="stroke:none" x="5.5" y="230.730876623" font-family="Helvetica" font-size="12" font-weight="normal" font-style="italic">TiDB-Lightning Tool Set</text>
```

**Pull request merged**